cli/src/commands/workspace/add.rs

```diff
@@ -50,6 +52,7 @@ enum SparseInheritance {
 #[derive(clap::Args, Clone, Debug)]
 pub struct WorkspaceAddArgs {
     /// Where to create the new workspace
+    #[arg(add = ArgValueCompleter::new(PathCompleter::dir()))]
```

Comment:
Do you know if this is different from `value_hint = clap::ValueHint::DirPath` (which we use elsewhere)?

Reply:
So I use `fish` and both seem to work? I'm not sure what `PathCompleter` does differently

The [docs around `ValueHint`](https://docs.rs/clap/latest/clap/enum.ValueHint.html) do seem to have a warning about fish not supporting positional arguments but I can complete arbitrary paths so maybe there's no difference :man_shrugging: 

I only used the `PathCompleter` because that's what my search turned up, but happy to switch this to `ValueHint::DirPath` if anyone has any preference towards that

Reply:
Will be addressed in https://github.com/jj-vcs/jj/pull/7337